src/scripts/main.js

```diff
@@ -1,3 +1,109 @@
 'use strict';
 
-// write code here
+const TBODY = document.querySelector('.field tbody');
+const ROWS = TBODY.rows;
```

Comment:
Code smell: Similar to the `TBODY` variable, `ROWS` is also a global variable that can cause problems. It's better to get rows inside the functions that need it.